content/en/security/threats/workload_security_rules/custom_rules.md

```diff
@@ -42,78 +97,69 @@ For more information, see [CSM Threats Detection Rules][7].
 
 ## Create a custom Agent rule
 
-You can create an individual custom Agent rule, deploy it as a [new Agent policy](#deploy-the-policy-in-your-environment), and reference it in a [custom detection rule](#create-a-custom-detection-rule).
-
-1. On the [**Agent Configuration**][4] page, select **New Rule**, and then select **Manual rule creator**.
-2. Add a name and description for the rule.
-3. In **Expression**, define the Agent expression using Datadog Security Language (SECL) syntax.
-
-    {{< img src="security/cws/workload_security_rules/define_agent_expression.png" alt="Adding a rule to the Expression field" >}}
-
-    For example, to monitor for suspicious container clients:
-
-    ```text
-    exec.file.path in ["/usr/bin/docker", "/usr/local/bin/docker",
-    "/usr/bin/kubectl", "/usr/local/bin/kubectl"] && container.id != ""
-    ```
-
-4. Click **Create Agent Rule**. This automatically navigates you back to the **Agent Configuration** page.
-
-After you create a custom Agent rule, the change is saved along with other pending rule updates. To apply the change to your environment, [deploy the updated custom policy to the Agent](#deploy-the-policy-in-your-environment).
-
-## Deploy the policy in your environment
+You can create a custom Agent rule and deploy it as part of a new Agent policy. Later, when defining a custom [detection rule][3], you reference the custom Agent rule and add expression parameters.
 
-Custom Agent rules are deployed to the Agent in a custom policy separate from the default policy. The custom policy contains custom Agent rules as well as [default rules that have been disabled](#disable-default-agent-rules).
+1. Go to [Security > Cloud Security Managment > Agent Configuration][3].
+2. Create or open a policy.
+3. In **Actions**, select **Manual rule creator**.
+4. Add a name and description for the rule.
+5. In **Expression**, define the Agent expression using [Datadog Security Language (SECL) syntax][16].
+6. Click **Create Agent Rule**. This automatically navigates you back to the policy page.
 
-You can use Remote Configuration to automatically deploy the custom policy to your designated hosts (all hosts or a defined subset of hosts), or manually upload it to the Agent on each host.
+After you create a custom Agent rule, the change is saved along with other pending rule updates. To apply the change to your environment, deploy the updated custom policy to the Agent.
 
 ### Remote Configuration
 
-1. On the **Agent Configuration** page, click **Deploy Agent Policy**.
-2. Select **Remote Configuration**.
-3. Choose whether to **Deploy to All Hosts** or **Deploy to a Subset of Hosts**. To deploy the policy to a subset of hosts, specify the hosts by selecting one or more service tags.
-4. Click **Deploy**.
+To perform remote configuration, you use the Datadog UI to apply policies to infrastructure. When you enable a policy, it is applied to the infrastructure identified by the policy's tags.
```

Comment:
Another point of discussion we can have at some time that isn't this time: is there a meaningful difference between "you use" and "you can use," or should we pick one and stick with it?